tests/plugins/dialogui/manual/required/required.md

```diff
@@ -2,12 +2,16 @@
 @bender-ui: collapsed
 @bender-ckeditor-plugins: wysiwygarea, toolbar, dialog, dialogui, image, table, link, iframe, floatingspace
 
+1. Switch on a screen reader.
```

Comment:
These new test steps indicate that the test should be only performed on screen readers and may result in skipping tests by developers. It would be good to split test steps for screen readers and other browsers.